3.10/arangosearch-ranking.md

```diff
@@ -53,6 +53,67 @@ Scoring functions cannot be used outside of `SEARCH` operations, as the scores
 can only be computed in the context of a View, especially because of the
 inverse document frequency (IDF).
 
+### Stable pagination for results
+
+The `SORT` operation does not guarantee a stable sort if there is no unique value
+to sort by. This leads to an undefined order when sorting equal documents.
+
+To achieve stable pagination, you must meet the following requirements: 
+- the dataset should not be changed
+- the `SORT` operation must have at least one field with an unique value to sort by
+
+When stable sort is required, you can use a tiebreaker field. If the application
+has a preferred field that indicates the order of documents with the same score,
+then this field should be used in the `SORT` operation as a tiebreaker. If there
+is no such field, you can use the `_id` system attribute as it is unique and
+present in every document. 
+
+```aql
+FOR doc IN viewName
+  SEARCH ANALYZER(STARTS_WITH(doc._fulltext, 'movie name'), 'text_en')
+  SORT BM25(doc) DESC, doc._id
+  LIMIT 10, 20
+  RETURN doc
+```
+
+Based on the [IMDB movie dataset](arangosearch-example-datasets.html#imdb-movie-dataset)
+and using a [View definition](arangosearch-fulltext-token-search.html#view-definition),
+you can run this query:
+
+```aql
+FOR doc IN imdb
+  SEARCH ANALYZER(doc.description IN TOKENS("ninja", "text_en"), "text_en")
+  LET score = BM25(doc)
+  SORT score DESC
+  LIMIT 0, 5
+  RETURN { title: doc.title, score }
+```
+
+| title | score |
+|:------|:-------|
+| Red Shadow: Akakage | 8.882122039794922 |
+| Beverly Hills Ninja | 7.128915786743164 | 
+| Naruto the Movie: Ninja Clash in the Land of Snow | 7.041049957275391 | 
+| TMNT | 7.002012729644775 | 
+| Batman Begins | 6.30634880065918 | 
+
+If you change the query to `LIMIT 5, 5`, then you get:
+
+| title | score |
+|:------|:-------|
+| Batman Begins | 6.30634880065918 | 
+| Shogun Assassin  | 5.8539886474609375 | 
+| Scooby-Doo and the Samurai Sword | 5.736422538757324 | 
+| Revenge of the Ninja | 5.212964057922363 | 
+| Winners & Sinners 2: My Lucky Stars | 5.165824890136719 | 
+
+Note how the position of Batman Begins is changed. It has the same score as
+Teenage Mutant Ninja Turtles II: The Secret of the Ooze.
```

Comment:
We did not see the TMNT result with the score so far! And I would also mention that Batman Begins appears twice, which is the result of the unstable sort, but it wouldn't hurt to also explain the outcome from a user PoV.